### PR TITLE
Return original JSON from getrawtransaction RPC

### PR DESCRIPTION
**Pull request by romanz**
Should fix https://github.com/romanz/electrs/issues/427

### CODE REVIEW

File: src/daemon.rs [hunk 1]
```diff
@@ -113,9 +114,13 @@ impl Daemon {
         &self,
         txid: &Txid,
         blockhash: Option<BlockHash>,
-    ) -> Result<json::GetRawTransactionResult> {
+    ) -> Result<Value> {
+        let args = blockhash.map_or_else(
```
Comment (Kixunil) [inline src/daemon.rs:118]: TBH, I find `match` clearer than `map_or_else`, the additional boilerplate is minimal.

**Review by Kixunil — APPROVED** [review]
LGTM

File: src/daemon.rs (hunk 1 above)
Comment (romanz) [reply src/daemon.rs]: Agree, but it seems that passing `null` blockhash works as well :)
https://github.com/bitcoin/bitcoin/blob/f8b20fd35b0eb42d28afb302a070e5d9bc21a99f/src/rpc/rawtransaction.cpp#L180

**Comment by romanz** [comment]
Tested locally on signet:
```
 echo '{"jsonrpc":"2.0","method":"blockchain.transaction.get","params":["170271bb6ba3678a3e014a79d7c3d51a59273739857b6fc2f6b933018c8ad445",true],"id":123}' | netcat localhost 60601 | jq .
{
  "id": 123,
  "jsonrpc": "2.0",
  "result": {
    "blockhash": "0000007a53005f3380efcf2aa321f59783e2ade331f4f22e8c8da31efe764304",
    "blocktime": 1626439369,
    "confirmations": 17,
    "hash": "9048e07b91984144b3f351339a7a32b91bc5da670cd1e526b42f7b6680a60a81",
    "hex": "02000000000101d83ef4111c649dd18f1ff86e280112fdfc1a39fbcfb64521debb6bb35e670d8c0100000000fdffffff023ae0f505000000002251200031b8688da76b2874923eb412bcf5421dad27669f64c7a2efcbbc8e3468f14300e1f505000000002251201f15c64dba6971c1ed1234927e2741dd6ce4190c8db95bdd5104dcc8a569359901400beb91c3cfad0e74813a826bca86560f4b9f2c6cd3d04317a8f86c7502ee938134703b90caa036a276cf5ceb7bb30c2a21510dae69a7f039210895f58541675acab60000",
    "in_active_chain": true,
    "locktime": 46794,
    "size": 205,
    "time": 1626439369,
    "txid": "170271bb6ba3678a3e014a79d7c3d51a59273739857b6fc2f6b933018c8ad445",
    "version": 2,
    "vin": [
      {
        "scriptSig": {
          "asm": "",
          "hex": ""
        },
        "sequence": 4294967293,
        "txid": "8c0d675eb36bbbde2145b6cffb391afcfd1201286ef81f8fd19d641c11f43ed8",
        "txinwitness": [
          "0beb91c3cfad0e74813a826bca86560f4b9f2c6cd3d04317a8f86c7502ee938134703b90caa036a276cf5ceb7bb30c2a21510dae69a7f039210895f58541675a"
        ],
        "vout": 1
      }
    ],
    "vout": [
      {
        "n": 0,
        "scriptPubKey": {
          "address": "tb1pqqcms6yd5a4jsayj866p9084ggw66fmxnajv0gh0ew7gudrg79ps3yaqpf",
          "asm": "1 0031b8688da76b2874923eb412bcf5421dad27669f64c7a2efcbbc8e3468f143",
          "hex": "51200031b8688da76b2874923eb412bcf5421dad27669f64c7a2efcbbc8e3468f143",
          "type": "witness_v1_taproot"
        },
        "value": 0.99999802
      },
      {
        "n": 1,
        "scriptPubKey": {
          "address": "tb1pru2uvnd6d9curmgjxjf8uf6pm4kwgxgv3ku4hh23qnwv3ftfxkvsax5kxz",
          "asm": "1 1f15c64dba6971c1ed1234927e2741dd6ce4190c8db95bdd5104dcc8a5693599",
          "hex": "51201f15c64dba6971c1ed1234927e2741dd6ce4190c8db95bdd5104dcc8a5693599",
          "type": "witness_v1_taproot"
        },
        "value": 1
      }
    ],
    "vsize": 154,
    "weight": 616
  }
}
```

File: src/daemon.rs (hunk 1 above)
Comment (Kixunil) [reply src/daemon.rs]: Ah, so `None` gets serialized as `null`, right?

Comment (romanz) [reply src/daemon.rs]: Yep :)

**Pull request merged**